pkg/tsdb/elasticsearch/response_parser_frontend_test.go

```diff
@@ -1569,29 +1562,7 @@ func TestLogsAndCount(t *testing.T) {
 	})
 
 	t.Run("level field", func(t *testing.T) {
-		// FIXME: config datasource with messageField=<unset>, levelField="level"
-		result, err := queryDataTest(query, response)
-		require.NoError(t, err)
-
-		require.Len(t, result.response.Responses, 1)
-		frames := result.response.Responses["A"].Frames
-		require.True(t, len(frames) > 0)
-
-		requireFrameLength(t, frames[0], 2)
-		fieldMap := make(map[string]*data.Field)
-		for _, field := range frames[0].Fields {
-			fieldMap[field.Name] = field
-		}
-
-		// require.Contains(t, fieldMap, "level") // FIXME
-		// field := fieldMap["level"]
-
-		// requireStringAt(t, "debug", field, 0)
-		// requireStringAt(t, "error", field, 1)
-	})
-
-	t.Run("level field remap", func(t *testing.T) {
-		// FIXME: config datasource with messageField=<unset>, levelField="fields.lvl"
+		// message field = line, level field = lvl
```

Comment:
Here we had 2 tests where we were testing if `level` field is correctly set - if the field's name is `level` and `lvl`. But we already have remap test in place in https://github.com/grafana/grafana/blob/main/pkg/tsdb/elasticsearch/response_parser_test.go#L1046. This way, we don't have to make `queryDataTest` more complicated and the test coverage stays the same.